app/views_redesigned/properties/_standard_card.html.erb

```diff
@@ -0,0 +1,36 @@
+<%= card do %>
+  <%= card_header do %>
+
+    <%= card_title add_class: "mb-0" do %>
+      <div class="d-flex justify-content-between align-items-center">
+        <h6 class="mb-0">
+          <%= tag.span "", class: "fas fa-circle small mr-1 text-#{property_status_color(property.status)}",
+            title: property_status_title(property.status), data: tooltip_expando %>
+          <%= link_to property.name, property, class: "link-muted" %>
+        </h6>
+        <div class="position-relative">
+          <%= link_to property_property_screenshots_path(property), method: :put, class: "btn btn-xs btn-icon btn-light transition-3d-hover" do %>
+            <span class="fas fa-camera btn-icon__inner"></span>
+          <% end if authorized_user.can_admin_system? %>
+          <%= link_to property.url, class: "btn btn-xs btn-icon btn-light transition-3d-hover", target: "_blank" do %>
+            <span class="fas fa-external-link btn-icon__inner"></span>
+          <% end %>
+        </div>
+      </div>
+    <% end %>
+  <% end %>
+
+  <%= card_body do %>
+    <%= link_to image_tag(property.screenshot.attached? ? property.screenshot : "missing-screenshot.jpg", class: "img-fluid"), property %>
```

Comment:
Let's not wrap this in `card_body` so it can be full width. See [here](https://github.com/gitcoinco/code_fund_ads/blob/0185565413cbee92aacb7d2568693e6246aa1ec8/app/views_redesigned/campaigns/_standard_card.html.erb#L10) for an example.

Reply:
Sounds good. Made the change.